### PR TITLE
Selection Sort

### PR DESCRIPTION
**Pull request by Chiadikaobixo**
Selection sort is a sorting algorithm, whereby you place the smaller values into sorted position one at a time.
https://en.wikipedia.org/wiki/Selection_sort

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/Solidity/pull/27"></a>

### CODE REVIEW

File: src/Sorts/SelectionSort.sol [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title returns an array of sorted numbers
+ * @author Chiadikaobi https://github.com/Chiadikaobixo
```
Comment (Panquesito7) [inline src/Sorts/SelectionSort.sol:6]: We should also use this syntax in another PR. Will be making an issue for it soon. 🙂
```suggestion
 * @author [Chiadikaobi](https://github.com/Chiadikaobixo)
```

**Comment by Chiadikaobixo** [comment]
Hi @mkubdev kindly look into this.

File: src/Sorts/SelectionSort.sol [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+/**
+ * @title returns an array of sorted numbers
+ * @author [Chiadikaobi](https://github.com/Chiadikaobixo)
+ * @dev https://en.wikipedia.org/wiki/Selection_sort
+ */
+
+/** @dev Selection sort is a sorting algorithm, whereby you place 
+ * the smaller values into sorted position one at a time.
+*/
+contract SelectionSort{
```
Comment (mkubdev) [inline src/Sorts/SelectionSort.sol:13]:
```suggestion
contract SelectionSort {
```

**Review by mkubdev — CHANGES_REQUESTED** [review]
Hi! Thanks for your contribution ✨
The sort algorithm looks good to me, there is space for optimization also.

Do you know about [Foundry](https://book.getfoundry.sh/getting-started/first-steps)? We encourage folks to write the Foundry test alongside the PR. You can see one example [here](https://github.com/TheAlgorithms/Solidity/blob/main/src/Sorts/test/BubbleSort.t.sol)

Have a good day!

**Comment by Chiadikaobixo** [comment]
I will definately look into it. must i write test for this selection sort before it will be merged?

**Comment by mkubdev** [comment]
> I will definately look into it. must i write test for this selection sort before it will be merged?

We can merge it, and open an issue to add the test later. :)

**Review by mkubdev — APPROVED** [review]
LGTM 🚀

**Pull request merged**